### DISCUSSION TITLE
Support 2c2p Gateway

### DISCUSSION THREAD

**Discussion by Uhtredtb** [inspireui/support#5744] [Request new Native Payment]
Hi, please make native support for 2C2P gateway.

**Comment by Uhtredtb** [comment]
Hi, there also 2c2p flutter plugin release can you convert this and add it to the app

https://pub.dev/packages/pgw_sdk

**Comment by geraltofrivia86** [comment]
+1 We need this too

**Comment by DominicToret** [comment]
I also want to see this gateway become native too
